org.opentosca.container.core/src/org/opentosca/container/core/next/trigger/SituationListener.java

```diff
@@ -125,5 +149,25 @@ void situationBeforeUpdate(final Situation situation) {
             this.sitTrigInstRepo.add(newInstances);
         }
     }
+    
+    private void sendServiceInstanceAdaptationEvent(SituationsMonitor monitor) {
+        final Map<String,Object> eventProperties = Maps.newHashMap();
+        eventProperties.put("SERVICEINSTANCE", monitor.getServiceInstance());
+        //eventProperties.put("SITUATIONS", monitor.getSituations());
+        eventProperties.put("NODE2SITUATIONS", monitor.getNode2Situations());
+        
+        
+        Event situationAdaptationEvent = new Event("org_opentosca_situationadaptation/requests", eventProperties);         
+        this.getEventAdminService().postEvent(situationAdaptationEvent);
+    }
+    
+    
+    
+    private EventAdmin getEventAdminService() {        
+        BundleContext ctx = org.opentosca.container.core.Activator.getContext();
+        ServiceReference<?> ref = ctx.getServiceReference(EventAdmin.class.getName());
+        EventAdmin eventAdmin = (EventAdmin)ctx.getService(ref);
+        return eventAdmin;
```

Comment:
Why is this not bound to the component through a Component.xml?

Reply:
Because we remove those in the osgi less branch right?